docs/data/date-pickers/date-time-picker/CustomDateTimePicker.tsx

```diff
@@ -1,57 +1,32 @@
 import * as React from 'react';
-import dayjs, { Dayjs } from 'dayjs';
+import dayjs from 'dayjs';
 import AlarmIcon from '@mui/icons-material/Alarm';
 import SnoozeIcon from '@mui/icons-material/Snooze';
-import TextField from '@mui/material/TextField';
 import ClockIcon from '@mui/icons-material/AccessTime';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
-import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
-import { MobileDateTimePicker } from '@mui/x-date-pickers/MobileDateTimePicker';
+import { Unstable_NextDateTimePicker as NextDateTimePicker } from '@mui/x-date-pickers/NextDateTimePicker';
 import Stack from '@mui/material/Stack';
 
 export default function CustomDateTimePicker() {
-  const [dateWithNoInitialValue, setDateWithNoInitialValue] =
-    React.useState<Dayjs | null>(null);
-  const [dateWithInitialValue, setDateWithInitialValue] =
-    React.useState<Dayjs | null>(dayjs('2019-01-01T18:54'));
-
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
       <Stack spacing={3}>
-        <DateTimePicker
+        <NextDateTimePicker
           openTo="hours"
```

Comment:
This no longer makes sense as it does nothing—the demo does not work at all.
It also sort of uncovers an issue that the time section is not focused when clicking the open icon.

Reply:
Prop removed, indeed make no sense

Reply:
Side note: We could add a warning when `openTo` equals a view that has no UI